.gitignore

```diff
@@ -5,6 +5,16 @@
 *.so
 *.dylib
 
+# Example binaries (build artifacts from examples/)
```

Comment:
Do not modify this file, and by default ignore the submission of such files in subsequent commits.